cmd/rpc/admin_handlers.go

```diff
@@ -0,0 +1,758 @@
+		}
+		// Create and return the transaction to be sent
+		return types.NewVotePollTransaction(p, ptr.PollJSON, ptr.PollApprove, s.config.NetworkID, s.config.ChainId, s.controller.ChainHeight(), ptr.Fee)
+	})
+}
+
+// ConsensusInfo retrieves node consensus information
+func (s *Server) ConsensusInfo(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
+	if err := r.ParseForm(); err != nil {
+		write(w, err, http.StatusBadRequest)
+		return
+	}
+	summary, err := s.controller.ConsensusSummary()
+	if err != nil {
+		write(w, err, http.StatusInternalServerError)
+		return
+	}
+	w.Header().Set(ContentType, ApplicationJSON)
+	w.WriteHeader(http.StatusOK)
+	if _, e := w.Write(summary); e != nil {
+		s.logger.Error(e.Error())
+	}
+}
+
+// PeerInfo retrieves node peer information
+func (s *Server) PeerInfo(w http.ResponseWriter, _ *http.Request, _ httprouter.Params) {
+	peers, numInbound, numOutbound := s.controller.P2P.GetAllInfos()
+	write(w, &peerInfoResponse{
+		ID:          s.controller.P2P.ID(),
+		NumPeers:    numInbound + numOutbound,
+		NumInbound:  numInbound,
+		NumOutbound: numOutbound,
+		Peers:       peers,
+	}, http.StatusOK)
+}
+
+// PeerBook retrieves the node's peer book
+func (s *Server) PeerBook(w http.ResponseWriter, _ *http.Request, _ httprouter.Params) {
+	write(w, s.controller.P2P.GetBookPeers(), http.StatusOK)
+}
+
+// Config retrieves the node's configuration file
+func (s *Server) Config(w http.ResponseWriter, _ *http.Request, _ httprouter.Params) {
+	write(w, s.config, http.StatusOK)
+}
+
+// txHandler is a helper function that abstracts common workflows of sending transactions.
+// It takes an HTTP response writer, an HTTP request, and a callback function that performs
+// transaction-specific logic using the private key and transaction request data.
+func (s *Server) txHandler(w http.ResponseWriter, r *http.Request, callback func(privateKey crypto.PrivateKeyI, ptr *txRequest) (lib.TransactionI, error)) {
+	// Create a new transaction request object.
+	ptr := new(txRequest)
+
+	// Parse and validate the incoming HTTP request JSON, unmarshalling its body into ptr.
+	if ok := unmarshal(w, r, ptr); !ok {
+		return
+	}
+
+	// Initialize a new keystore from the server's configured data directory.
+	keystore, ok := newKeystore(w, s.config.DataDirPath)
+	if !ok {
+		return
+	}
+
+	// Update the transaction request with address information derived from the nickname, if available.
+	getAddressFromNickname(ptr, keystore)
+
+	// Determine the signer address; use the supplied address if signer is unspecified.
+	signer := ptr.Signer
+	if len(signer) == 0 {
+		signer = ptr.Address
+	}
+
+	// Retrieve the private key for the signer from the keystore using the provided password.
+	privateKey, err := keystore.GetKey(signer, ptr.Password)
+	if err != nil {
+		write(w, err, http.StatusBadRequest)
+		return
+	}
+	// Set the public key in transaction request for reference.
+	ptr.PubKey = privateKey.PublicKey().String()
```

Comment:
If you port over this from main

```go
operatorPrivateKey, err := keystore.GetKey(ptr.Address, ptr.Password)
	if err != nil {
		write(w, err, http.StatusBadRequest)
		return
	}
	ptr.PubKey = operatorPrivateKey.PublicKey().String()
	p, err := callback(signerPrivateKey, ptr)
	if err != nil {
		write(w, err, http.StatusBadRequest)
		return
	}
```

Then your conflict is likely non-existent in the other file